conda-recipe/vcdat/meta.yaml

```diff
@@ -20,4 +20,4 @@ source:
   git_rev: v0.0.1
```

Comment:
@doutriaux1 that would be greatly appreciated. Then we can always have a working master/nightly. Thank you